tests/unit/amo/components/TestUserCollection.js

```diff
@@ -57,6 +57,22 @@ describe(__filename, () => {
     );
   });
 
+  it('can render a collection that is loading', () => {
+    const props = {
+      authorId: 1234,
+      id: 1,
+      name: 'collection name',
+      // When a collection is in a loading state, numberOfAddons is null.
+      numberOfAddons: null,
```

Comment:
That is not what I can read here: https://github.com/mozilla/addons-frontend/blob/7cfbafe7ddb2325850d2c73f1c395994e0d25bdf/src/amo/reducers/collections.js#L75

Reply:
Hmm, that's odd. I wonder why Flow wasn't catching that problem already. Here is where we set `numberOfAddons` to `null`: https://github.com/mozilla/addons-frontend/blob/7cfbafe7ddb2325850d2c73f1c395994e0d25bdf/src/amo/reducers/collections.js#L1076.